tasks/databases.rake

```diff
@@ -311,6 +417,22 @@ namespace :data do
   end
 end
 
+def print_memory_usage
+  memory_before = `ps -o rss= -p #{Process.pid}`.to_i
```

Comment:
i wonder if there is a OS agnostic way of doing this? this assumes linux-like systems. in addition, linux/unix have different `ps` commands so the results are different